v2/googlecloud-to-neo4j/docs/README.md

```diff
@@ -63,18 +63,20 @@ The template requires the following parameters:
 Template can be executed using the following gcloud command:
 
 ```sh
+export PROJECT=<my-project>
 export IMAGE_NAME_VERSION=b0.12
-export TEMPLATE_IMAGE_SPEC="gs://neo4j-dataflow/flex-templates/images/googlecloud-to-neo4j-image-spec-${IMAGE_NAME_VERSION}.json"
+export TEMPLATE_SPEC_GCSPATH="gs://neo4j-dataflow/flex-templates/images/googlecloud-to-neo4j-image-spec-${IMAGE_NAME_VERSION}.json"
 export REGION=us-central1
 export MACHINE_TYPE=n2-highmem-8
  
 gcloud dataflow flex-template run "googlecloud-to-neo4j-text-cli-`date +%Y%m%d-%H%M%S`" \
-    --template-file-gcs-location="$TEMPLATE_IMAGE_SPEC" \
+    --project "$PROJECT" \
     --region "$REGION" \
+    --template-file-gcs-location="TEMPLATE_SPEC_GCSPATH" \
     --parameters jobSpecUri="gs://neo4j-dataflow/job-specs/testing/text/text-northwind-jobspec.json" \
     --parameters neo4jConnectionUri="gs://neo4j-dataflow/job-specs/testing/connection/auradb-free-connection.json" \
-    --max-workers=1 \
-    --worker-machine-type=${MACHINE_TYPE} 
+    --parameters workerMachineType="$WORKER_MACHINE" \
+    --max-workers=1
```

Comment:
We want to rely more and more in the generated doc to avoid it going out of date.

https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/main/v2/googlecloud-to-neo4j/README_Google_Cloud_to_Neo4j.md

Perhaps you can work a bit with the "example" annotations to make it similar to what you see here.